client/users/friends-list.tsx

```diff
@@ -327,36 +327,42 @@ function VirtualizedFriendRequestsList({ height }: { height: number }) {
     useUserEntriesSelector(outgoingRequests),
     areUserEntriesEqual,
   )
-  const sortedIncoming = useMemo(() => sortUserEntries(incomingUserEntries), [incomingUserEntries])
-  const sortedOutgoing = useMemo(() => sortUserEntries(outgoingUserEntries), [outgoingUserEntries])
+  const incomingUserIds = useMemo(
```

Comment:
Why do this work at all? You're already mapping over this resulting array anyway, why not just make that map work on the entries instead of IDs?